scripts/configure-installer-tests-cluster-ppc64le.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/bash
```

Comment:
@sarveshtamba Lets not duplicate script unnecessarily. I can see this file is a copy paste of ```configure-installer-tests-cluster-s390x.sh```, instead you can do it in a better way. For example - separate out the common lines like ```oc import-image nodejs...``` in another script and call the script in ```configure-installer-tests-cluster.sh``` file.

Reply:
Are you suggesting to delete `configure-installer-tests-cluster-s390x.sh` which itself is a duplicate of `configure-installer-tests-cluster.sh` in the first place?

Reply:
@sarveshtamba Let be like this. later it will be east to do the clean of thing if needed in a separate pr. Thanks for your patience

Reply:
/lgtm